gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/BundleMapper.java

```diff
@@ -0,0 +1,26 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import java.util.List;
+import java.util.UUID;
+
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.UriType;
+
+public class BundleMapper {
+
+    private static void bundleMapper() {
```

Comment:
1. this method should return a bundle
2. It is not actually a mapper - it is not mapping anything to a bundle. It creates a bundle